README.md

```diff
@@ -14,7 +14,7 @@ npm install @mapcolonies/export-interfaces
 export-interfaces exposes the 'Export Manager' interface with common functions to be implement
 ```typescript
 export interface IExportManager {
-  exportTask: (data: ExportTaskRequest<TaskParameters>) => Promise<ExportTaskResponse>;
+  createExportTask: (data: CreateExportTaskRequest<TaskParameters>) => Promise<CrateExportTaskResponse>;
   getGeometries: (jobId: string) => Promise<TaskGeometry[]>;
```

Comment:
Why do you need "getGeometries" method if "CreateExportTaskRequest" returns the "Geometries", what is the scenario that needs them?

Reply:
i just forgot change the readme file , fixed